cmake/FindONNX.cmake

```diff
@@ -0,0 +1,31 @@
+ocv_clear_vars(HAVE_ONNX)
+
+set(ORT_INSTALL_DIR "" CACHE PATH "ONNX Runtime install directory")
```

Comment:
`INSTALL_DIR` is usually used to specify location where to install something.

Consider renaming to `ONNX_ROOT_DIR` (do not use `ONNX_DIR` / `ONNX_ROOT` as they are reserved by CMake).

Reply:
Should I change it right now or after having a custom builder working?

Reply:
Added extra variable into custom builder:
- `-DONNXRUNTIME_ROOT_DIR=/opt/onnxrt`